spec/models/form_submission_attempt_spec.rb

```diff
@@ -164,46 +131,17 @@
       end
 
       it 'sends a received email' do
-        notification_email = double
-        allow(notification_email).to receive(:send)
-        allow(SimpleFormsApi::NotificationEmail).to receive(:new).with(
-          config,
-          notification_type:,
-          user_account: anything
-        ).and_return(notification_email)
+        allow(SimpleFormsApi::Notification::SendNotificationEmailJob).to receive(:perform_async)
         form_submission_attempt = create(:form_submission_attempt)
 
         form_submission_attempt.vbms!
 
-        expect(notification_email).to have_received(:send)
-      end
-
-      context 'form_data is nil' do
-        let(:config) do
-          {
-            form_data: nil,
-            form_number: anything,
-            date_submitted: anything,
-            lighthouse_updated_at: anything,
-            confirmation_number: anything
-          }
-        end
-
-        it 'gracefully handles the nil form_data' do
-          notification_email = double
-          allow(JSON).to receive(:parse)
-          allow(SimpleFormsApi::NotificationEmail).to receive(:new).with(
-            config,
-            notification_type:,
-            user_account: anything
-          ).and_return(notification_email)
-          allow(notification_email).to receive(:send)
-          form_submission_attempt = create(:form_submission_attempt)
-
-          form_submission_attempt.vbms!
-
-          expect(JSON).to have_received(:parse).with('{}')
-        end
+        expect(SimpleFormsApi::Notification::SendNotificationEmailJob).to have_received(:perform_async).with(
+          notification_type:,
+          form_submission_attempt:,
+          form_number: anything,
+          user_account: anything
+        )
```

Comment:
I'm deleting tests in this file because we no longer need to reach into `SimpleFormsApi::NotificationEmail`. We're interacting only with `SimpleFormsApi::Notification::SendNotificationEmailJob` instead.